classes/class-wc-connect-api-client.php

```diff
@@ -291,13 +291,15 @@ protected function request( $method, $path, $body = array() ) {
 				return $headers;
 			}
 
+			$http_timeout = 60; // 1 minute
+			wc_set_time_limit( $http_timeout + 10 );
```

Comment:
Since WC could be deactivated independently of us, please test for the function before calling it